internal/Confusion.ecl

```diff
@@ -17,17 +21,16 @@ Classify_Result:= Core_Types.Classify_Result;
 Confusion_Detail := Core_Types.Confusion_Detail;
 
 /**
-  * Generate the confusion matrix, to compare actual versus predicted response
-  * variable values.
-  * @param dependents the original response values.
-  * @param predicts the predicted responses.
-  * @return confusion matrix in Confusion_Detail format.
-  * @see ML_Core.Types.Confusion_Detail
-  */
+ * Detail confusion records to compare actual versus predicted response
+ * variable values.
+ * @param dependents the original response values
+ * @param predicts the predicted responses
+ * @return confusion counts by predicted and actual response values.
+ */
```

Comment:
Why did you remove @see ...?  That clarifies the interface.  Also the old @return specified the return type, which was removed in the new comment.